modules/ocf_kubernetes/manifests/master/loadbalancer.pp

```diff
@@ -69,31 +69,10 @@
   # nginx requests and send them into the cluster.
   #
   # TODO: If we expose TCP services, we may need to add more.
-  #
-  # IPv6 addresses have to be specified separately as they cannot be in the vrrp
-  # packet together (keepalived 1.2.20+) so they need to be in a
-  # virtual_ipaddress_excluded block instead.
-  $virtual_addresses = [
-    # Primary load balancer IP (v4)
-    '169.229.226.79',
-  ]
-  $virtual_addresses_v6 = [
-    # Primary load balancer IP (v6)
-    '2607:f140:8801::1:79',
-  ]
-  $keepalived_secret = lookup('kubernetes::keepalived::secret')
-
-  package { 'keepalived':; } ->
-  file { '/etc/keepalived/keepalived.conf':
-    content => template('ocf_kubernetes/master/loadbalancer/keepalived.conf.erb'),
-    mode    => '0400',
-  } ~>
-  service { 'keepalived': }
-
-  $vip = 'lb'
-
-  class { 'ocf_kubernetes::master::loadbalancer::ssl':
-    vip => $vip,
+  class { 'ocf_lb':
+    vip_names                => ['lb'],
+    keepalived_secret_lookup => 'kubernetes::keepalived::secret',
+    vrid                     => 51,
```

Comment:
Why make this dynamic if only one value is used after all that? I'd just leave it out for now, hardcode it in the template, and refactor later if you do need the complexity of having multiple `vrid`s

Reply:
I added this after I tried to add this class in the ceph module and ran into issues. It's necessary to avoid that if we have more than one VIP anywhere (which is kind of the whole point).

Reply:
Ah I see, I didn't know you were planning to use this for ceph, that makes more sense then, keep it like this.